helm/openwhisk/values.yaml

```diff
@@ -161,6 +163,7 @@ nginx:
   httpPort: 80
   httpsPort: 443
   httpsAdminPort: 8443
+  httpsNodePort: 31001
```

Comment:
This is fine to add as a default, but I need to fix the usage in nginx-svc.yaml to also check for ingress.type being NodePort.  I will do a PR for that fix today.  Right now we depend on the absence of a value for httpsNodePort when using ingress.type of standard.

Reply:
I have made other changes from the review. I will wait for your changes to be merged in before progressing on this one further.